content/geometry/geometry-index.adoc

```diff
@@ -0,0 +1,58 @@
+= Vehicle structure
+:toc:
+:toclevels: 4
+
+include::vehicle-structure/Grp_Vehicle_Part_keyword_vehicle_part_idx.adoc[leveloffset=+1]
+include::vehicle-structure/Grp_Exterior.adoc[leveloffset=+2]
+include::vehicle-structure/Grp_Exterior_Dynamic.adoc[leveloffset=+3]
```

Comment:
Reference to Grp_Exterior_Static is missing.

Reply:
Reference added.